tools/bin/go_core_ccip_deployment_tests

```diff
@@ -6,22 +6,23 @@ SCRIPT_PATH=`dirname "$0"`; SCRIPT_PATH=`eval "cd \"$SCRIPT_PATH\" && pwd"`
 OUTPUT_FILE="../output.txt"
 USE_TEE="${USE_TEE:-true}"
```

Comment:
we don't need this anymore?

Reply:
282d22f2e21aab12762fd8ada27b03af7a698df8